docker/test/develop.yml

```diff
@@ -0,0 +1,70 @@
+# This needs at least compose 1.6.0
+version: '2'
+
+services:
+  mysqldata:
+    image: mysql:5.7
+    command: /bin/true
+
+  mysql:
+    image: mysql:5.7
+    volumes_from:
+      - mysqldata
+    ports:
```

Comment:
Can we remove the port mapping? It causes problems when mysql is running natively on the docker host.